Library/Application Support/Code/User/keybindings.json

```diff
@@ -9,7 +9,6 @@
         "command": "-extension.vim_ctrl+d",
         "when": "editorTextFocus && vim.active && !inDebugRepl"
     },
-
```

Comment:
All the newlines were removed by the JSON formatter, which is now automatically triggered on save (see settings change below).